src/topology/algebra/big_operators/ring.lean

```diff
@@ -0,0 +1,243 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl
+-/
+import topology.algebra.big_operators.basic
+import topology.algebra.ring
+
+/-!
+# Infinite sum in a ring
+
+This file provides lemmas about the interaction of infinite sums and (scalar) multiplication.
```

Comment:
I think the motivation is clearer as:
```suggestion
This file provides lemmas about the interaction between infinite sums and multiplication.

Some results about scalar multiplication are also provided.
```
(assuming you keep the `smul` in here at all.